src/app/services/graph-client/msal-service.ts

```diff
@@ -72,6 +72,20 @@ export function logOut() {
   msalApplication.logout();
 }
 
+export function logOutPopUp() {
+  // @ts-ignore
```

Comment:
Why are we ignoring Typescript here

Reply:
The methods I am accessing are private and surprisingly accessible as long as you ignore typescript.

It's an allowed workaround until the team releases a logoutPopUp implementation

Reply:
You can add this as a comment, so that we know to stop ignoring Typescript in the future.